generators/artifacthub/package_test.go

```diff
@@ -48,7 +48,7 @@ func TestGetChartUrl(t *testing.T) {
 				}
 				byt, _ := json.MarshalIndent(comp, "", "")
 
-				f, err := os.Create(dirName + "/" + comp.Kind + ".json")
+				f, err := os.Create(dirName + "/" + comp.Component.Kind + ".json")
```

Comment:
Should we be mandating json?

Reply:
Perhaps this is something to create a separate issue about.

Reply:
We don't use this anymore (which was added by Abhi Kumar).
This is leftover to be deleted.

Reply:
deleted

Reply:
comment at wrong place.